tests/flow/new_react/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -865,11 +865,7 @@ var ReactClass = React.createClass({
   render: function(): any {
     // Any state access here seems to make state any
     this.state;
-    return (
-      <div>
-        {this.state.bar.qux}
-      </div>
-    );
+    return <div>{this.state.bar.qux}</div>;
```

Comment:
I've run both variations over the following open source projects:

* https://github.com/andrewngu/sound-redux
* https://github.com/WebbyLab/itsquiz-wall
* https://github.com/echenley/react-news/

Allowing elements with a single expression to be inlined generally resulted in less changes than always expanding.